tests/python/gaia-ui-tests/gaiatest/apps/camera/app.py

```diff
@@ -115,6 +152,39 @@ def tap_switch_source(self):
         Wait(self.marionette).until(lambda m: not current_camera_mode == self.camera_mode)
         self.wait_for_capture_ready()
 
+    def tap_switch_to_front_camera(self):
+	self.marionette.find_element(*self._camera_switch_to_front_locator).tap()
+	switch_to_rear_button = Wait(self.marionette).until(expected.element_present(*self._camera_switch_to_rear_locator))
+        Wait(self.marionette).until(expected.element_displayed(switch_to_rear_button))
+        
+    def tap_switch_to_rear_camera(self):
+	self.marionette.find_element(*self._camera_switch_to_rear_locator).tap()
+	switch_to_front_button = Wait(self.marionette).until(expected.element_present(*self._camera_switch_to_front_locator))
+        Wait(self.marionette).until(expected.element_displayed(switch_to_front_button))
+		
+    def tap_menu_button(self):
+	self.marionette.find_element(*self._menu_button_locator).tap()
+	menu_opened = Wait(self.marionette).until(expected.element_present(*self._close_menu_locator))
+	Wait(self.marionette).until(expected.element_displayed(menu_opened))
+	
+    def tap_enable_timer(self):
```

Comment:
Agreed. Also this function does more than it's name implies. I would split it into:
```python
def tap_enable_timer_for(self, length_in_seconds=5):
    self._tap_enable_timer()
    self._choose_length(length_in_seconds)
    self._wait_for_timer_icon_in_view_finder()
```